Sources/Internal/Version.swift

```diff
@@ -16,5 +16,41 @@ import Foundation
 
 @available(macOS 10.15, iOS 13, tvOS 13, watchOS 6, *)
 struct Version {
-  static let version = "11.3.0-beta"
+  static let sdkVersion = "11.3.0-beta"
+
+  // returns value of form gl-PLATFORM_NAME/PLATFORM_VERSION
+  static func platformVersionHeader() -> String {
+    let version = ProcessInfo.processInfo.operatingSystemVersion
+    let versionString = "\(version.majorVersion).\(version.minorVersion).\(version.patchVersion)"
+    #if os(iOS)
+      return "gl-ios/\(versionString)"
+    #elseif os(watchOS)
+      return "gl-watchos/\(versionString)"
+    #elseif os(tvOS)
+      return "gl-tvos/\(versionString)"
+    #elseif os(macOS)
+      return "gl-macos/\(versionString)"
+    #else
+      return ""
+    #endif
+  }
+
+  // returns the build time major version of swift
+  static func swiftMajorVersion() -> String {
```

Comment:
Xcode version provides more information - including the Swift version - see https://github.com/firebase/firebase-ios-sdk/blob/main/FirebaseCore/Sources/FIRFirebaseUserAgent.m#L92 and https://swiftversion.net/

Reply:
This isn't useful. It prints the Xcode build number =>  "DTXcodeBuild": 16A231a. That makes it difficult to map. 

If we need exact version, I'll just need to expand this function to include minor Swift versions.

Reply:
Expanded the function to include major.minor. Leaving patch out for now.